lib/osf-components/addon/components/file-actions-menu/template.hbs

```diff
@@ -48,5 +48,17 @@
                 {{t 'general.share'}}
             </dd.trigger>
         </SharingIcons::Dropdown>
+        <OsfLink
+            data-test-rename-button
+            data-analytics-name='Rename file'
+            aria-label={{t 'file_actions_menu.rename'}}
+            local-class='RenameLink'
```

Comment:
Can we use the `DropdownItem` class?

Reply:
Yes, this is now updated.

Reply:
Ok, that sounds good. Let me test that change.

Reply:
This now works as a Button and is also now using the DropdownItem class for styling.